src/DynamoCoreWpf/NodeViewCustomization/NodeViewCustomizationLoader.cs

```diff
@@ -48,6 +48,14 @@ public static INodeViewCustomizations LoadCustomizations(Assembly assem, ILogger
             return new NodeViewCustomizations(types);
         }
 
+        private static IEnumerable<Type> GetCustomizationTypes(Assembly assem)
```

Comment:
Isn't this method the same as `GetCustomizationTypesUsingReflection` in `NodeModelAssemblyLoader`? Can we just have one function?

Reply:
@aparajit-pratap this function is the same except it doesn't use reflection - if you're okay relying on reflection everywhere I can use that one and put it somewhere both classes can access - what do you think?

Reply:
I'm not sure what you mean? Isn't doing `Assembly.GetTypes` reflection too?

Reply:
yes, you're right of course - sorry, the distinction I was trying to make was in how the two functions find the `INodeViewCustomization` type - one uses a direct reference to the loaded type - the other has to use Type.GetTypes() to search for the type or load it using a string.

Reply:
Ah, ok, gotcha